wb-codemod/transforms/migrate-color-to-tokens.ts

```diff
@@ -0,0 +1,136 @@
+        }
+
+        // Remove the source specifier from the import declaration
+        const hasRemainingSpecifiers = path.value.specifiers?.some(
+            (specifier) => specifier.local?.name !== sourceSpecifier,
+        );
+
+        // Replace default import with named import
+        const targetSpecifier = j.importSpecifier(
+            j.identifier(TARGET_SPECIFIER),
+        );
+
+        // If the target import is already used, we add a named import to the
+        // existing import declaration.
+        if (targetImport.size() > 0) {
+            // NOTE: We only add the specifier if it doesn't exist already.
+            const specifierExistsInTarget = targetImport
+                .get()
+                .node.specifiers.some(
+                    (specifier: ModuleSpecifier) =>
+                        specifier.local?.name === TARGET_SPECIFIER,
+                );
+
+            if (!specifierExistsInTarget) {
+                targetImport.get().node.specifiers.push(targetSpecifier);
+                // sort the specifiers
+                targetImport
+                    .get()
+                    .node.specifiers.sort(
+                        (a: ImportSpecifier, b: ImportSpecifier) =>
+                            a.imported.name.localeCompare(b.imported.name),
+                    );
+            }
+
+            // Remove the "wonder-blocks-color" import declaration
+            if (!hasRemainingSpecifiers) {
+                j(path).remove();
+            } else {
+                // Remove the default specifier from the import declaration
+                path.value.specifiers = path.value.specifiers?.filter(
+                    (specifier) => specifier.local?.name !== sourceSpecifier,
+                );
+            }
+            return;
+        }
+
+        // The target import doesn't exist, so we create a new one.
+        const newTargetImport = j.importDeclaration(
+            [targetSpecifier],
+            j.literal(TARGET_IMPORT_DECLARATION),
+        );
+
+        // Color was the only specifier in the import declaration.
+        if (!hasRemainingSpecifiers) {
+            // Replace the existing import declaration with the new one.
+            j(path).replaceWith(newTargetImport);
+        } else {
+            // There are other specifiers in the import declaration (e.g. mix,
+            // fade).
+            // We keep the other specifiers and remove the default specifier.
+            path.value.specifiers = path.value.specifiers?.filter(
+                (specifier) => specifier.local?.name !== sourceSpecifier,
+            );
+            // Move the default specifier to a named import in the target import
+            // declaration
+            j(path).insertAfter(newTargetImport);
+        }
+    });
+
+    // Step 3: Replace the usage
+    root.find(j.Identifier, {name: sourceSpecifier}).forEach((path) => {
+        path.node.name = TARGET_SPECIFIER;
+    });
```

Comment:
Is this to maintain existing uses of `import {color} from "@khanacademy/wonder-blocks-tokens"`?